File: jdroid-android-firebase-admob/src/main/java/com/jdroid/android/firebase/admob/AdsStats.java
package com.jdroid.android.firebase.admob;


import com.jdroid.java.date.DateUtils;

public class AdsStats {
	
	private static Long lastInterstitialOpenedTimestamp = 0L;
	
	public static void onInterstitialOpened() {
		lastInterstitialOpenedTimestamp = DateUtils.nowMillis();
	}
	
	public static Long getLastInterstitialOpenedTimestamp() {
		return lastInterstitialOpenedTimestamp;
	}
}

File: jdroid-android-firebase-admob/src/main/java/com/jdroid/android/firebase/admob/AdsStats.kt
package com.jdroid.android.firebase.admob

import com.jdroid.java.date.DateUtils

object AdsStats {

    private var lastInterstitialOpenedTimestamp: Long = 0L

    fun onInterstitialOpened() {
        lastInterstitialOpenedTimestamp = DateUtils.nowMillis()
    }

    fun getLastInterstitialOpenedTimestamp(): Long {
        return lastInterstitialOpenedTimestamp
    }
}
